pixi.js.d.ts

```diff
@@ -659,32 +665,32 @@ declare namespace PIXI {
         static TEMP_MATRIX: Matrix;
 
     }
-    export class ObservablePoint {
 
-        constructor(cb: () => any, scope?: any, x?: number, y?: number);
+    class PointLike {
```

Comment:
I agree, I think using `PointLike` everywhere instead of `Point | ObservablePoint` would be better, I was just incorporating @clark-stevenson suggestion since he's the project owner. :)